app/views/shared/_error_messages.html.haml

```diff
@@ -1,3 +1,6 @@
 - if object.errors.any?
   .flash-message.alert#error_explanation
     %strong= t('generic.validation_errors', count: object.errors.count)
+- object.errors[:antispam].each do |error|
```

Comment:
Can we just use `:base` here? I think that's where we add general errors on other models, so could be useful everywhere. By the way, you're creating two elements with the same ID on the same page.